### DISCUSSION TITLE
iOS/platform-independent version?

### DISCUSSION THREAD

**Discussion by henningko** [krzyzanowskim/STTextView#33] [Questions]
Thank you for putting together such a great tool! Since TextKit2 is available for both iOS and macOS, I am wondering whether you plan on expanding support to iOS for STTextView? Haven't found a TextKit2 based implementation there, and it's a struggle dealing with UITextView.

**Comment by krzyzanowskim** [comment]
While the text layout part could be shared to some degree, the editing part is very much OS-dependent as each OS offers a different API. I hope to find time to work on iOS version, as I'd really like to make it happen.

**Comment by krzyzanowskim** [comment]
FYI the iOS and macOS support is currently on `main` branch and will be released as v2
